packages/sdk/src/client.ts

```diff
@@ -93,6 +93,10 @@ export function SoundClient({
         return await editionContract.supportsInterface(interfaceIds.ISoundEditionV1)
       } catch (err) {
         onError(err)
+        const chainId = await _getNetworkChainId()
```

Comment:
what happens when `await _getNetworkChainId` itself fails

Reply:
this look ok? https://github.com/soundxyz/sdk/pull/123/commits/29ee8fb7dff3afff7a50a9aa262bb0a24859e785